src/bundle/Resources/views/themes/admin/ui/component/tag_view_select_selected_item.html.twig

```diff
@@ -0,0 +1,17 @@
+{% set can_delete = can_delete is defined ? can_delete : true %}
```

Comment:
I think that `can_delete` is a little bit confusing and may be mistaken for user permissions.
```suggestion
{% set is_deletable = can_delete is defined ? can_delete : true %}
```